source/agora/cli/client/DefaultProcess.d

```diff
@@ -11,9 +11,9 @@
 
 *******************************************************************************/
 
-module agora.cli.DefaultProcess;
+module agora.client.DefaultProcess;
 
-import agora.cli.CLIResult;
+import agora.client.Result;
```

Comment:
Right, actually, this won't compile either, the changes to the module name need to be at the same time as the file path change.

It might be simpler to just merge all commits.

Reply:
@Geod24 I fixed it